tests/config/config.TEMPLATE.pgsql.inc.php

```diff
@@ -100,3 +100,8 @@
 
 [debug]
 show_stacktrace = On
+
+[queues]
+
+default_queue = null
```

Comment:
Is there a practical case for this to have another value?

Reply:
Yes, if you're using a Message broker, you could put add the default queue here.

Reply:
I've changed it to use as value the string "test_default_queue".